test/e2e/webdriver/index.js

```diff
@@ -67,6 +73,7 @@ async function setupFetchMocking(driver) {
       return { cancel: true };
     }
     function clone(obj) {
+      console.log(`clone`);
```

Comment:
Still need to remove this?

Reply:
sorry, I thought I committed this change but I lost it somewhere along the branch switching 😟